playwright.config.js

```diff
@@ -21,7 +21,7 @@ module.exports = defineConfig({
     userAgent: 'Playwright',
   },
   /* set the expect timeout to 10s */
-  expect: { timeout: 10000 },
```

Comment:
📓 I increased this because running these tests locally sometimes take a little longer. And, we're tracking the duration so we don't care if it takes longer.